### ISSUE TITLE
Generated audio roughly an octave lower than input?

### ISSUE THREAD

**Issue by CASE-WESTERN** [rncm-prism/prism-samplernn#31]
Simple sounding issue, but one I can't trace. 

I'm getting some great generated results, but they are all about an octave to 16 notes lower than what the input, as best as I can tell by playing with the clips in a DAW.

I have set everything that I can to a sample rate of 16000, which is correct for my source audio. Other than the occasional clip exporting nothing but a high-pitched tone, it seems to be working great. 

Basically I get whale songs until I speed them up, at which point they're spot-on.

**Comment by relativeflux** [comment]
@CASE-WESTERN  Hi, thanks for getting in touch, and thank you for informing me about this issue. I'm not sure what else could be causing this other than an issue with the sample rate. Are you sure the input audio is at the same sample rate as the output? Does this issue occur with audio generated during training, or separately using the `generate.py` script? Although the former does use code from the latter. Also did you use the `chunk_audio.py` script to create your dataset? I've heard reports of odd things happening when using a dataset consisting of chunks created by other means.

Could you post your training script command call, with parameters?

**Comment by CASE-WESTERN** [comment]
Well after going through everything again and scratching my head it looks like I've correctly accounted for a 16000Hz sample rate... but had uploaded the original 44100Hz file for chunk_audio.py, not the version I had downsampled. Fixed.

Thanks again and cheers!

**Comment by relativeflux** [comment]
@CASE-WESTERN No problem, glad you found the fix!
